standalone/install.ts

```diff
@@ -14,9 +14,29 @@ function download(data: Data): Promise<Data> {
 			console.log(chalk.yellow("Binary already downloaded, skipping..."));
 			return resolve(data);
 		}
-
 		console.log(chalk.yellow(`Downloading Pact Standalone Binary v${PACT_STANDALONE_VERSION} for platform ${data.platform} from ${data.url}`));
 
+		// Track downloads through Google Analytics unless testing or don't want to be tracked
+		if (!process.env.DO_NOT_TRACK) {
+			console.log(chalk.gray("Please note: we are tracking this download anonymously to gather important usage statistics. " +
+			"To disable tracking, set 'DO_NOT_TRACK=true' as an environment variable."));
+			const CI = ["CI", "CONTINUOUS_INTEGRATION"].some((key) => process.env[key] !== undefined);
+			request.post({
```

Comment:
I think this is the issue with #91, it should fail gently with perhaps a note at `DEBUG` level that it didn't send any metrics.